### PR TITLE
Update Cartographics map functionality. DHH-848

### PR DESCRIPTION
**Pull request by VivianChu**

### CODE REVIEW

**Comment by VivianChu** [comment]
@ucsdlib/developers - please review, comments, etc... Thanks

File: app/views/layouts/ucsdlib.html.erb [hunk 1]
```diff
@@ -5,8 +5,9 @@
 		<%= render :partial => 'layouts/twitter_card' %>
 		<link href="<%= opensearch_catalog_path(:format => 'xml', :only_url => false) %>" title="<%= application_name%>" type="application/opensearchdescription+xml" rel="search"/>
 
-	  <script src="//libraries.ucsd.edu/jwplayer/jwplayer.js"></script>
-	  <script src="//libraries.ucsd.edu/jwplayer/key.js"></script>
+        <script src="//libraries.ucsd.edu/jwplayer/jwplayer.js"></script>
+        <script src="//libraries.ucsd.edu/jwplayer/key.js"></script>
+        <script src="https://www.mapquestapi.com/sdk/leaflet/v2.2/mq-map.js?key=zrnNkNZDkELMvViepTqFjZP8bEKxjDaG"></script>
```
Comment (mcritchlow) [inline app/views/layouts/ucsdlib.html.erb]: should this key be in an environment variable?

Comment (VivianChu) [inline app/views/layouts/ucsdlib.html.erb]: @mcritchlow I tried to use the env variable, however when I did the view source of the object, people still can see the key.  Attached is the screen shot.
mapquestkey

Comment (mcritchlow) [inline app/views/layouts/ucsdlib.html.erb]: that's a very good point. the main benefit i can see of having it as an ENV variable (but perhaps not in Ansible vault) is if we ever need to change keys we might expect to look there first. But, whatever you think.

Comment (VivianChu) [inline app/views/layouts/ucsdlib.html.erb]: I'm fine with either way.  For env variable, let me ask John to add it.

Comment (VivianChu) [inline app/views/layouts/ucsdlib.html.erb]: @mcritchlow The env variable has been added.

**Comment by mcritchlow** [comment]
@ucsdlib/developers we still need feedback on this PR, please.

**Comment by hweng** [comment]
@VivianChu Is that possible to have a test to reflect this change?

**Comment by VivianChu** [comment]
@hweng The test has been added.  Thanks

**Comment by hweng** [comment]
:+1:

**Comment by lsitu** [comment]
👍

**Pull request merged**